File: 프로그래머스/lv1/86051. 없는 숫자 더하기/없는 숫자 더하기-1.java
import java.util.Arrays;
import java.util.stream.IntStream;
class Solution {
	public int solution(int[] numbers) {
		return IntStream.rangeClosed(0, 9)
						// noneMatch() : 주어진 모든 요소들이 조건을 만족하지 않으면 반환
						.filter(idx -> Arrays.stream(numbers).noneMatch(number -> number == idx))
						.sum();
	}
}

File: 프로그래머스/lv1/86051. 없는 숫자 더하기/없는 숫자 더하기-1.js
function solution(numbers) {
  return Array.from({ length: 10 }, (_, idx) => idx) // 길이 10, 인덱스 0~9까지의 배열 생성
    .filter((number) => !numbers.includes(number)) // 위 생성된 인덱스와 numbers 의 요소들 비교 후 0~9까지 없는 숫자만 필터링
    .reduce((acc, curr) => acc + curr); // 필터링된 요소들을 더함
}
